newdle/client/src/components/creation/FinalStep.js

```diff
@@ -123,35 +139,51 @@ export default function FinalStep() {
           color="violet"
           type="submit"
           disabled={!canSubmit}
-          onClick={createNewdle}
+          onClick={isEditing ? editNewdle : createNewdle}
           loading={submitting}
         >
-          Create your Newdle!{' '}
-          <span role="img" aria-label="Newdle">
-            🍜
-          </span>
+          {!isEditing ? (
+            <span>
+              Create your Newdle!
+              <span role="img" aria-label="Newdle">
+                {' 🍜'}
+              </span>
+            </span>
+          ) : (
+            'Confirm'
+          )}
         </Button>
       </div>
-      <div className={styles['link-row']}>
-        <Button
-          size="small"
-          color="violet"
-          basic
-          onClick={() => dispatch(setStep(1))}
-          disabled={submitting}
-          icon="angle double left"
-          content="Change participants"
-        />
-        <Button
-          size="small"
-          color="violet"
-          basic
-          onClick={() => dispatch(setStep(2))}
-          disabled={submitting}
-          icon="angle left"
-          content="Change time slots"
-        />
-      </div>
+      {!isEditing && (
+        <div className={styles['link-row']}>
+          <Button
+            size="small"
+            color="violet"
+            basic
+            onClick={() => dispatch(setStep(STEPS.PARTICIPANTS))}
+            disabled={submitting}
+            icon="angle double left"
+            content="Change participants"
+          />
+          <Button
+            size="small"
+            color="violet"
+            basic
+            onClick={() => dispatch(setStep(STEPS.TIMESLOTS))}
+            disabled={submitting}
+            icon="angle left"
+            content="Change time slots"
+          />
+        </div>
+      )}
     </Container>
   );
 }
+
+FinalStep.propTypes = {
+  isEditing: PropTypes.bool,
+};
+
+FinalStep.defaultProps = {
+  isEditing: true,
```

Comment:
that seems like a rather strange default tbh. I'd default to false or make it required

Reply:
Agree, I expect false too (as I've made the others), this was a mistake.